hikari/impl/bot.py

```diff
@@ -640,7 +640,7 @@ async def close(self) -> None:
             return
 
         self._has_aborted = True
-        self._guild_chunker.close()
+        await self._guild_chunker.close()
```

Comment:
Mind reverting this? Im aware its a bug, but this will get fixed with #121 (just fixes the merge conflicts there might be)

Reply:
See line 473 :)
```
        calls = [
            ("rest", self._rest.close()),
            ("chunker", self._chunker.close()),
            ("voice handler", self._voice.close()),
            *((f"shard {s.id}", s.close()) for s in self._shards.values()),
        ]

        for coro in asyncio.as_completed([handle(*pair) for pair in calls]):
            await coro
```